app/assets/stylesheets/pageflow/themes/legacy/indicators.scss

```diff
@@ -0,0 +1,62 @@
+.entry {
+  .indicator {
+    color: #e6e6e6;
```

Comment:
Color literals like `#e6e6e6` should only be used in variable declarations; they should be referred to via variable everywhere else.